src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please include this file in your .gitignore as we are not using XML based configuration.